test/core/extensions/crypto_extension_test.cpp

```diff
@@ -395,12 +447,40 @@ TEST_F(CryptoExtensionTest, DISABLED_Secp256k1RecoverUncompressed) {
   ASSERT_EQ(ptrsize, res);
 }
 
+/**
+ * @given initialized crypto extensions
+ * @and a damaged secp256k1 signature and message
+ * @when call recovery public secp256k1 uncompressed key
+ * @then error code is returned
+ */
+TEST_F(CryptoExtensionTest, Secp256k1RecoverUncompressedFailure) {
+  WasmPointer sig = 1;
+  WasmPointer msg = 10;
+  WasmSpan res = WasmResult(20, 20).combine();
+  auto &sig_input = secp_signature;
+  auto &msg_input = secp_message_hash;
+  auto sig_buffer = Buffer(sig_input);
+  sig_buffer[4] = 0;  // damage signature
```

Comment:
```suggestion
  sig_buffer[4] = ~sig_buffer[4];  // damage signature
```
Because `sig_buffer[4]` can be already is zero, and signature will not damaged.

Reply:
It's not zero, since I assigned it beforehand, but ok, I will use it.

Reply:
Conversation resolved, but I did not see changes about it.
Pls, don't resolve conversation before take any decision and push commit with possible changes. Anyway, to comment if you reject sugession or disagree of that.

Reply:
I find zeroizing more appropriate and it works as it should, since I use predefined signature buffer.

Reply:
Ok.